application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +48,12 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            message.delete().flatMap(any -> dmUser(message)).queue(res -> {
+            }, new ErrorHandler().handle(ErrorResponse.CANNOT_SEND_TO_USER, err -> {
+                warnUser(message).queue((res) -> {
+                    res.delete().queueAfter(1, TimeUnit.MINUTES);
+                });
```

Comment:
```suggestion
                warnUser(message).queue(res -> res.delete().queueAfter(1, TimeUnit.MINUTES));
```
name it something better as well

Reply:
Done